recipes/autodiff/meta.yaml

```diff
@@ -39,7 +39,6 @@ about:
   home: https://autodiff.github.io
   license: MIT
   license_family: MIT
-  license_file: LICENSE
```

Comment:
Why did you remove this? Not including it in the package is a violation of the licence which states:

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

Reply:
Hmm I thought that was only needed if the `LICENSE` file is inside the `recipe` dir.

Reply:
Yeah if you put `[ci skip]` in the commit message it will skip it

Reply:
I'll do this then because I guess the issue with osx_64 is now solved (well, skipped - the `test_run.sh` exits if `CLANGXX` is defined!).

Reply:
Oh and it's also picked up from the work directory, i.e. the directory used for the build step containing the source. Conda-build might pick it up by default in this case, but explicitly listing its filename will make the CI fail if it is missing rather than risking it being missed.

Reply:
OK, all has passed (except the travis build that is still pending).